classes/image_types_tegra.bbclass

```diff
@@ -25,13 +25,13 @@ def tegra_dtb_extra_deps(d):
     deps = []
     if d.getVar('PREFERRED_PROVIDER_virtual/dtb'):
         deps.append('virtual/dtb:do_populate_sysroot')
-    if d.getVar('TEGRA_UEFI_DB_KEY') and d.getVar('TEGRA_UEFI_DB_CERT'):
+    if d.getVar('USE_UEFI_SIGNED_FILES'):
```

Comment:
The definition of this variable is is in `tegra-uefi-signing.bbclass`, and will, presumably, also be set in any custom signing bbclass, so the signing class needs to be inherited here in `image_types_tegra` too.

Also, this check, plus the other python references to this variable, need to explicitly compare to `"true"`.  Just using `d.getVar()` will always succeed if the variable is defined, even if set to `"false"`.

And now that the usage of this variable spans multiple classes, it might be good to add a `TEGRA_` prefix to it, just to avoid possible conflicts with other layers.

Reply:
Oh, and it probably wouldn't hurt to add a `??=` setting (to `"false"`) in this bbclass for this variable also, so it parses properly even if the signing class variable setting is null.